telegram/ext/filters.py

```diff
@@ -103,15 +103,13 @@ def __repr__(self):
             self.name = self.__class__.__name__
         return self.name
 
-    def filter(self, update):
+    def filter(self, message):
         """This method must be overwritten.
 
-        Note:
-            If :attr:`update_filter` is false then the first argument is `message` and of
-            type :class:`telegram.Message`.
-
         Args:
-            update (:class:`telegram.Update`): The update that is tested.
+            message (:class:`telegram.Message`): The message that is tested. If
+            :attr:`update_filter` is True update is `update` of type
```

Comment:
you mean, message is `update` of type update right? And you missed a comma there ;P